critiquebrainz/ws/review/views.py

```diff
@@ -23,6 +23,15 @@ def review_entity_handler(review_id):
     :statuscode 404: review not found
 
     :resheader Content-Type: *application/json*
+
+    .. code-block:: bash
```

Comment:
You should put these before status codes and error. It should be part of the description.